ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-sno-ranGen.yaml

```diff
@@ -49,12 +49,12 @@ spec:
             type: "fluentd"
             fluentd: {}
     - fileName: MachineConfigSctp.yaml
-      policyName: "mc-sctp-policy"
+      policyName: "config-policy"
```

Comment:
Why do we have a policy for this. It should be applied as part of the day 1 extra manifests, delete.

Reply:
Agreed. Removing them from the templates is being done under PR #834.